lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -226,24 +215,4 @@ private Optional<ErrorResponse> validateSessionParams(Map<String, String> sessio
         }
         return Optional.empty();
     }
-
-    @Tracing
-    private ClientSessionDetailsDto generateClientSessionDetails(
```

Comment:
What do you think? But, we can keep this method and create new ClientOauthSessionItem and then can use that. I thought, so, we don't lost history. But, I am not sure. Because, only using with govukSigninJourneyId.